ui/apps/platform/cypress/helpers/request.js

```diff
@@ -19,6 +19,7 @@
  * @returns Record<string, { method: string, url: string }>
  */
 export function getRouteMatcherMapForGraphQL(opnames) {
+    /** @type Record<string, { method: string, url: string }> */
```

Comment:
Prevents type errors from this JS file from manifesting in TS files that consume this function.